admin/users/views.py

```diff
@@ -132,47 +124,30 @@ def get_object(self, queryset=None):
 class ResetPasswordView(OSFAdmin, FormView):
     form_class = EmailResetForm
     template_name = 'users/reset.html'
-
-    def get(self, request, *args, **kwargs):
-        try:
-            return super(ResetPasswordView, self).get(request, *args, **kwargs)
-        except AttributeError:
-            return page_not_found(
-                request,
-                AttributeError(
-                    'User with id "{}" not found.'.format(
-                        self.kwargs.get('guid')
-                    )
-                )
-            )
-
-    def post(self, request, *args, **kwargs):
-        try:
-            return super(ResetPasswordView, self).post(request, *args, **kwargs)
-        except AttributeError:
-            return page_not_found(
-                request,
-                AttributeError(
-                    'User with id "{}" not found.'.format(
-                        self.kwargs.get('guid')
-                    )
-                )
-            )
+    context_object_name = 'user'
 
     def get_context_data(self, **kwargs):
         user = User.load(self.kwargs.get('guid'))
         try:
             self.initial.setdefault('emails', [(r, r) for r in user.emails])
         except AttributeError:
```

Comment:
Does the change in language from the latest commit address your concerns? @mfraezz https://github.com/CenterForOpenScience/osf.io/pull/5686/commits/a8eb2f16399f4ddf22f6b34b95ee4594301edd49